webapp/components/survey/CategoryDetails/LevelDetails/ItemDetails.js

```diff
@@ -1,55 +1,71 @@
 import React, { useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
 import * as R from 'ramda'
 
```

Comment:
Components always go to their own folder

Reply:
ok, sorry